server/docker/Dockerfile

```diff
@@ -36,4 +36,4 @@ RUN unzip patavi.zip && mv patavi-*/server/* .
 
 RUN cd public && yarn
 
-CMD ["pm2", "start", "--no-daemon", "server.js"]
+CMD ["node", "server.js"]
```

Comment:
I think this should not be changed, as this is the 'normal' dockerfile for patavi

Reply:
Agree